launch/launch/actions/execute_process.py

```diff
@@ -622,7 +636,6 @@ def connection_made(self, transport):
             )
             super().connection_made(transport)
             self.__process_event_args['pid'] = transport.get_pid()
```

Comment:
@Blast545 I think we should update this in the same place in which we set the `_subprocess_transport`, no?

Reply:
Also, I think the protocol no longer needs to know about the `__action`.

Reply:
Addressed both comments with 0516975 and c20dab4 respectively